internal/pkg/scorecard/helpers/versions.go

```diff
@@ -38,15 +38,8 @@ func ValidateVersion(version string) error {
 
 }
 
-func IsV1alpha2() bool {
-	if viper.Sub("scorecard").GetString(VersionOpt) == v1alpha2 {
-		return true
-	}
-	return false
-}
-
-func IsLatestVersion() bool {
-	if viper.Sub("scorecard").GetString(VersionOpt) == LatestScorecardVersion {
+func IsV1alpha2(version string) bool {
+	if version == v1alpha2 {
```

Comment:
Nit: This could be simplified to `return version == v1alpha2`

Reply:
agreed, made the change.